azure-pipelines/build-pr.yml

```diff
@@ -215,3 +215,10 @@ jobs:
           wait_for_completion: 'True'
           pytest_mark: ''
           max_run_duration: '30m'
+
+  # Now run the build for the Data Selection sub-folder
+  - job: BuildDataSelection
+    pool:
+      vmImage: 'ubuntu-18.04'
+    steps:
+      - template: build_data_quality.yaml
```

Comment:
Not sure we want to couple those two pipelines? We have a problem with long build time already, adding more bits on will make it even slower.

Reply:
It is actually a separate job that runs in parallel of all the other stages. It only takes 6 mins to run i.e. is one of the first to finish. Do you think this is a problem?

Reply:
Resolved offline